__tests__/articles.test.js

```diff
@@ -0,0 +1,89 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+require("jest-sorted");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/articles/:article_id", () => {
+  test("200: responds with an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toBeInstanceOf(Object);
+
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
We should test for an actual number here since we know what should come back 🙂

Reply:
changed this 1 to mirror the test, thanks for helping :)